packages/framer-motion/cypress/integration/drag.ts

```diff
@@ -234,9 +234,9 @@ describe("Drag", () => {
             .get("[data-testid='draggable']")
             .trigger("pointerdown", 10, 10)
             .trigger("pointermove", 15, 15)
-            .wait(50)
+            .wait(200)
             .trigger("pointermove", 300, 300, { force: true })
-            .wait(50)
+            .wait(200)
```

Comment:
Why is this required?

Reply:
The layout animation I added is like 100 - 200ms long so this delay has to be long enough for them to trigger and fuck up the measurements